docs/_docs/windows.md

```diff
@@ -81,6 +90,7 @@ the site generation process. It can be done with the following command:
 ```sh
 $ chcp 65001
 ```
+#### Time-Zone Management (Chocolatey method)
```

Comment:
Why???
Even after repeatedly saying, this is **not** specific to the Chocolatey method.. why???